CONTRIBUTING.md

```diff
@@ -1,6 +1,10 @@
-CONTRIBUTING.md
+# CONTRIBUTING.md
+
+## Table of Contents
+- [Claiming an Issue](#Claiming-an-Issue)
 
 ## Claiming an Issue
```

Comment:
I see what you're doing with the "Back to Top" Feature and it's good that said we can clean up this a little bit.

```suggestion
## Claiming an Issue [Back to Top](#Table-of-Contents)
```